assets/javascripts/modules/checkout/form-elements.js

```diff
@@ -1,36 +1,52 @@
 define(['$'], function ($) {
 
-    var formElements = {
-        $FIRST_NAME: $('.js-checkout-first'),
-        $LAST_NAME: $('.js-checkout-last'),
-        $EMAIL: $('.js-checkout-email'),
-        $ADDRESS1: $('.js-checkout-house'),
-        $ADDRESS2: $('.js-checkout-street'),
-        $ADDRESS3: $('.js-checkout-town'),
-        $POSTCODE: $('.js-checkout-postcode'),
-        $ACCOUNT: $('.js-checkout-account'),
+    return {
```

Comment:
I thought about it. I'd prefer to keep the selector definition static for to make it straight forward to refactor the form markup.